docker/s6/services/.s6-svscan/crash

```diff
@@ -1,7 +1,3 @@
 #!/bin/sh
 
-echo "Container crashed"
-
-echo "Trying to clean file cache first..."
-
-rm -rf /lazyraster/cache
\ No newline at end of file
+echo "Container crashed"
```

Comment:
It calls it "cras" but this is also called when you stop the container. So should probably say "container shutting down" or something less drastic.